proto/buf/connect/demo/eliza/v1/eliza.proto

```diff
@@ -50,3 +57,23 @@ message ConverseRequest {
 message ConverseResponse {
   string sentence = 1;
 }
+
+// IntroduceRequest describes the sentence said to the ELIZA program.
+message IntroduceRequest {
+  string sentence = 1;
+}
+
+// IntroduceResponse describes the sentence responded by the ELIZA program.
+message IntroduceResponse {
+  string sentence = 1;
```

Comment:
Would this just be a single sentence response for a stream of sentence requests?

Reply:
Yeah.  The stream would return a series of `IntroduceResponse` messages.